tests/plugins-tester/scripts/update-top-plugins-list.js

```diff
@@ -22,6 +22,9 @@ const fetchPluginsList = async ( url, filename ) => {
             pluginsList.push( plugin.slug );
         } );
 
+		// Sort the list
+		pluginsList.sort();
+
```

Comment:
lint 
(seems like the file uses spaces)

Reply:
Done